build/patches/Bromite-AdBlockUpdaterService.patch

```diff
@@ -983,12 +1508,54 @@ new file mode 100644
 +  tasks_.erase(task);
 +}
 +
++std::string AdBlockUpdaterService::GetAdBlockFiltersURL() {
++  return pref_service_->GetString(kAdBlockFiltersURL);
++}
++
++void AdBlockUpdaterService::SetAdBlockFiltersURL(const std::string url) {
++  pref_service_->SetString(kAdBlockFiltersURL, url);
++}
++
++std::string AdBlockUpdaterService::GetMostRecentIndexedVersion() {
++  auto version = ruleset_service_->GetMostRecentlyIndexedVersion();
++  return version.content_version;
++}
++
++long AdBlockUpdaterService::GetLastUpdate() {
```

Comment:
Perhaps we can document this function with a comment? Saying that it returns a timestamp in UNIX milliseconds

Reply:
there is already in `.h`